src/main/java/uk/ac/ebi/embl/gff3tools/exception/ValidationException.java

```diff
@@ -28,6 +28,15 @@ public ValidationException(String rule, int line, String message) {
         this.rule = rule;
     }
 
+    public ValidationException(int line, String message) {
```

Comment:
The rule is fairly important as it allows the user to toggle the behavior using the CLI or configuration file. We MUST provide it as feedback.